src/jesse_lib.erl

```diff
@@ -86,3 +88,22 @@ is_null(null) ->
 is_null(_Value) ->
   false.
 
+%% @doc Run the RE against the subject using the `re_options' from the jesse
+%% application environment. `{capture, none}' is always used.
+-spec re_run( Subject :: iodata() | unicode:charlist()
+            , RE :: iodata() | unicode:charlist()
+            ) -> match
+               | nomatch.
+re_run(Subject, RE) ->
+  re:run(Subject, RE, [{capture, none} | re_options()]).
+
+%% @doc Returns the base re options from jesse environment which will be used
+%% when running client-provided patterns. By default, that is `[ucp]' for the
+%% fullest compatibility matching unicode code points beyond ISO Latin-1 in
+%% `\w', `\s', and `\d', etc. Use `[unicode]' instead for better performance at
+%% the expense of non-ISO Latin-1 compatibility. See also notes on `unicode' vs.
+%% `ucp' at
+%% {@link https://www.erlang.org/doc/man/re.html#compile-2. re:compile/2}
+-spec re_options() -> list().
+re_options() ->
+  application:get_env(jesse, re_options, [ucp]).
```

Comment:
When `ucp` is enabled, shouldn't `unicode` be added explicitly as well? It's a bit not clear from the documentation.

Reply:
I think you're right that you need both:

```erlang
1> re:run(<<"fōô"/utf8>>, "^\\w+$", [{capture, none}, ucp, unicode]).
match
2> re:run(<<"fōô"/utf8>>, "^\\w+$", [{capture, none}, ucp]).
nomatch
```

I will update the default and docs.

Reply:
what makes me think that it might be nice to have at least some test that showcases this feature

Reply:
I agree, I will write some eunit tests for these new utility functions.

Reply:
Done! The last commit (388cfa6cb55e148d021e96cc5bdfb944c0e3e292) should address your concerns and suggestions. Thanks!